lisa/tools/dmesg.py

```diff
@@ -37,7 +37,8 @@ def _check_exists(self) -> bool:
 
     def get_output(self, force_run: bool = False) -> str:
         command_output = self._run(force_run=force_run)
-        return command_output.stdout
+        stdout = re.sub(self.__color_code_pattern, "", command_output.stdout)
```

Comment:
Please change it to use `util.filter_ansi_escape`.

Reply:
@squirrelsc Done in latest push. Please check.

Reply:
Please help refine existing code, to remove duplicate pattern.

Reply:
@squirrelsc Done in latest push. Please check